loadflow/loadflow-validation/src/main/java/com/powsybl/loadflow/validation/StaticVarCompensatorsValidation.java

```diff
@@ -177,4 +183,22 @@ private static boolean checkSVCsValues(String id, float p, float q, float v, flo
         return validated;
     }
 
+    private static float closestBound(float bound1, float bound2, float value) {
+        if (Float.isNaN(value)) {
+            return Float.NaN;
+        }
+        if (Float.isNaN(bound1)) {
+            return bound2;
+        }
+        if (Float.isNaN(bound2)) {
+            return bound1;
+        }
+        return Math.abs(value - bound1) < Math.abs(value - bound2) ? bound1 : bound2;
+    }
+
+    private static float setPoint(float reactivePowerSetpoint, float bMinVV, float bMaxVV, ValidationConfig config) {
+        return ValidationUtils.boundedWithin(-bMaxVV, -bMinVV, reactivePowerSetpoint, config.getThreshold())
```

Comment:
Indeed here, the setpoint must be checked against [Qmin, Qmax] where Qmin = bMinVV and Qmax = bMaxVV, and not [-Qmax, -Qmin].
There was a mistake in the initial specifications.

The sign convention of the reactive power setpoint is opposed to the sign convention on terminal.getQ().
It is positive if the SVC actually injects some reactive power in the grid.